src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -143,14 +151,14 @@ private int AnalyzeTemplate(FileInfo templateFilePath, FileInfo parametersFilePa
                 if (!templateFilePath.Exists)
                 {
                     logger.LogError("Invalid template file path: {templateFilePath}", templateFilePath);
-                    return 2;
+                    return ErrorInvalidFile;
```

Comment:
Can we add tests to test exit codes for the CLI? At least for the easy ones like invalid file/missing file?

Reply:
CLI does not have tests yet. I wanted to add a test proj, but was not sure if it was appropriate to do it in this PR or in a separate PR.

Reply:
I think it'd be appropriate to add a test project and some number of tests related to your work in this PR. That way we can the CLI test coverage.